components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -170,13 +187,13 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                 >
                     <span
                         id={'channelHeaderDropdownIconLeft_' + id}
-                        className={'fa fa-angle-left SubMenu__icon-left' + (hasSubmenu && !isMobile ? '' : '-empty' + (isMobile ? ' mobile' : ''))}
+                        className={'fa fa-angle-left SubMenu__icon-left' + (hasSubmenu && !isMobile && (direction === 'left') ? '' : '-empty' + (isMobile ? ' mobile' : ''))}
```

Comment:
3/5 prefer the use of 'classnames'

Reply:
Agreed, but that's outside the scope of this PR. We're going to refactor all this menu code at some point since it's not quite robust enough.